physics/continuous_trajectory_test.cpp

```diff
@@ -729,7 +729,7 @@ TEST_F(ContinuousTrajectoryTest, Serialization) {
   EXPECT_EQ(4, checkpoint.last_point_size());
 
   auto const trajectory_read =
-      ContinuousTrajectory<World>::ReadFromMessage(message);
+      ContinuousTrajectory<World>::ReadFromMessage(InfiniteFuture, message);
```

Comment:
Name the parameter whose value is `InfiniteFuture`; same below.

Reply:
Done, also in `ephemeris_test.cpp`.